### ISSUE TITLE
Add "needs-merge" as a synonym for "changes-approved"

### ISSUE THREAD

**Issue by danuker** [chevah/github-hooks-server#39]
Since the tag name is "needs-merge", I keep trying to use it as a keyword. It does not work, and it should.

**Comment by adiroiban** [comment]
Yes. We can have a configuration with a list of synonyms for each action.

But, at the same time, we should use GitHub UI so using the "needs-merge" keyword is not really needed.
